arches/app/models/tile.py

```diff
@@ -284,10 +295,18 @@ def check_for_missing_nodes(self):
         missing_nodes = []
         for nodeid, value in self.data.items():
             try:
-                node = models.Node.objects.get(nodeid=nodeid)
+                try:
+                    node = SimpleNamespace(**next((x for x in self.serialized_graph["nodes"] if x["nodeid"] == nodeid), None))
+                except:
+                    node = models.Node.objects.get(nodeid=nodeid)
                 datatype = self.datatype_factory.get_instance(node.datatype)
                 datatype.clean(self, nodeid)
                 if self.data[nodeid] is None and node.isrequired is True:
+                    try:
+                        node.cardxnodexwidget_set.all()
```

Comment:
I think we could store off the result from the call to `node.cardxnodexwidget_set.all()` and use it on lines 310 and 311.